### PR TITLE
[KF-425] temporal supports

### PR DESCRIPTION
**Pull request by 7eben**

### CODE REVIEW

File: kuflow-temporal-activity-kuflow/test/_test_workflow.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        # Complete Workflow
+        result = await workflow.execute_activity(
+            KuFlowSyncActivities.complete_process,
+            models_temporal.CompleteProcessRequest(request.processId),
+            start_to_close_timeout=timedelta(seconds=120),
+            retry_policy=RetryPolicy(maximum_interval=timedelta(seconds=30)),
+        )
+
+        print(f"Result: {result}")
```
Comment (jrpedrianes) [inline kuflow-temporal-activity-kuflow/test/_test_workflow.py:40]: Is this shown only one time? is the way the logs must be added in temporal python SDK?

**Review by jrpedrianes — CHANGES_REQUESTED** [review]

File: kuflow-temporal-activity-kuflow/kuflow_temporal_activity_kuflow/kuflow_async_activities.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+class KuFlowAsyncActivities:
+    def __init__(self, kuflow_client: KuFlowRestClient) -> None:
+        self._kuflow_client = kuflow_client
+        self.activities = [self.create_task_and_wait_finished]
+
+    @activity.defn
+    async def create_task_and_wait_finished(
+        self,
+        request: models_temporal.CreateTaskRequest,
+    ) -> None:
```
[the remark this answers is not in the document]
Comment (7eben) [reply kuflow-temporal-activity-kuflow/kuflow_temporal_activity_kuflow/kuflow_async_activities.py:18]: That is an async activity, the return type is void en JAVA or None in python

[the remark this answers is not in the document]
Comment (jrpedrianes) [reply kuflow-temporal-activity-kuflow/kuflow_temporal_activity_kuflow/kuflow_async_activities.py:18]: ahh it's true, we delete the result type. 👍

File: kuflow-temporal-activity-kuflow/test/_test_workflow.py (hunk 1 above)
Comment (7eben) [reply kuflow-temporal-activity-kuflow/test/_test_workflow.py]: I will change it

**Review by jrpedrianes — APPROVED** [review]

**Pull request merged**